### DISCUSSION TITLE
How to use built-in Themes?

### DISCUSSION THREAD

**Discussion by plachow** [havit/Havit.Blazor#467] [General]
Is there some way how to use your themes which are listed in drop down on your demo page? Cosmo, Lux, Materia and others. Drop down has some unique integer values for each Theme but how to use it?

I tried to change css name in Custom themed Bootstrap CSS build but with no luck.

Dík :)

**Comment by crdo** [comment]
Hi, there are no themes as part of our library. 
What is listed in the sidebar of the docs is just to demonstrate that you can use the library with any Bootstrap theme of your choice.
You simply take the compiled CSS of the theme you choose and replace the link in either index.html or _Host.cshtml/_Layout.cshtml as seen on https://havit.blazor.eu/#css-javascript

**Comment by hakenr** [comment]
The themes demonstrated in our documentation are taken from https://bootswatch.com (loaded dynamically from https://bootswatch.com/api/5.json). Just use them from https://bootswatch.com instead of our/CDN version of bootstrap.css.

(In fact, we provide one "theme" in our library, it is the Bootstrap CSS build included in the lib (`_content/Havit.Blazor.Components.Web.Bootstrap/bootstrap.css`) as a customized alternative to plain Bootstrap.css from CDN.)

**Comment by plachow** [comment]
Thank you. I thought that those themes are created by you. Didn't know about Bootswatch. 
I use your BS theme so I will add some switch for Bootswatch themes.

**Comment by hakenr** [comment]
Thanks for feedback. We adjusted the theme-switcher in the documentation to make is more obvious that those are bootswatch.com themes.
